governance/api.go

```diff
@@ -245,7 +245,29 @@ func (api *PublicGovernanceAPI) MyVotingPower() (float64, error) {
 }
 
 func (api *PublicGovernanceAPI) ChainConfig() *params.ChainConfig {
-	return api.governance.InitialChainConfig()
+	num := rpc.LatestBlockNumber
+	return api.chainConfigAt(&num)
+}
+
+func (api *PublicGovernanceAPI) ChainConfigAt(num *rpc.BlockNumber) *params.ChainConfig {
+	return api.chainConfigAt(num)
+}
+
+func (api *PublicGovernanceAPI) chainConfigAt(num *rpc.BlockNumber) *params.ChainConfig {
+	var blocknum uint64
+	if num == nil || *num == rpc.LatestBlockNumber {
+		blocknum = api.governance.BlockChain().CurrentHeader().Number.Uint64()
+	} else if *num == rpc.PendingBlockNumber {
+		blocknum = api.governance.BlockChain().CurrentHeader().Number.Uint64() + 1
```

Comment:
Unlike `ItemsAt` api, this api returns `number+1` block's changeConfig when it's pendingBlockNumber. 
Shouldn't it be the same?

Reply:
Right. fixed in ed9c679f